### PR TITLE
Hide the password field

### LINKED ISSUE #1233: [BUG] [SECURITY] Password of emailaccount is plaintext and visible
**Describe the bug**
The password of the account to access the mailbox is visible in plaintext in the webinterface (and therefore also in the database)

**To Reproduce**
1. Go to '/admin/paperless_mail/mailaccount/1/change/'
2. Scroll down to 'Password'

**Expected behavior**
A hidden/obscured representation (all asterisks or something)
And also stored as a hash in the database.

**Screenshots**
I do not think so ;)

**Webserver logs**
```
Irrelevant
```

**Relevant information**
 - Host OS of the machine running paperless: [e.g. Archlinux / Ubuntu 20.04]
 - Version [e.g. 1.0.0]
 - Installation method: [docker / bare metal]
 - Any configuration changes you made in `docker-compose.yml`, `docker-compose.env` or `paperless.conf`.

### PR DESCRIPTION
**Pull request by sbrunner**
Fix: https://github.com/jonaswinkler/paperless-ng/issues/1233

### CODE REVIEW

**Comment by yschroeder** [comment]
This is a duplicate of #39. We already imported your original PR. However, as we now have your attention we can as well use this one to review 🙂

File: src/paperless_mail/admin.py [hunk 1]
```diff
@@ -1,9 +1,25 @@
 from django.contrib import admin
+from django import forms
 from paperless_mail.models import MailAccount, MailRule
 
 from django.utils.translation import gettext_lazy as _
 
 
+class MailAccountAdminForm(forms.ModelForm):
+
+    """Metadata classes used by Django admin to display the form."""
```
Comment (bauerj) [inline src/paperless_mail/admin.py:10]: This seems a bit redundant with almost the same comment below.

Comment (alexander-bauer) [reply src/paperless_mail/admin.py:10]: There might be need for a docstring on both the outer and inner class, though I'm not sure which one should carry more detail.

**Comment by bauerj** [comment]
Thanks for rebasing on `dev`!

**Comment by bauerj** [comment]
Thanks for rebasing on `dev`!

**Review by alexander-bauer — APPROVED** [review]
Looks good to me.

**Review by bauerj — APPROVED** [review]

**Pull request merged**